### PR TITLE
feat(TMRX-1099): build section buckets slice

### PR DESCRIPTION
**Pull request by ibrahim-kurhan**
User Story

As a web reader
I want to see articles presented in this new slice
So that I can enjoy the most up to date reader experience

[TMRX-1099](https://nidigitalsolutions.jira.com/browse/TMRX-1099)

Acceptance Criteria

Build Section Buckets slice in Storybook using Newskit [Grid](https://newskit.co.uk/components/grid/) - does not need to be integrated/exported (consider using the [GridLayout](https://www.newskit.co.uk/components/grid-layout/) component)

Add data as Knobs in storybook so that design can try different Headline lengths or images etc. 

Understand how different font sizing will affect spacing and alignment and how it would be impacted on different browsers

Understand how the Newskit grid copes with different size screens from S, M, L, and XL

Highlight and document findings with/for Himesh

Pick this slice from the Figma design https://www.figma.com/file/phASjaJaTK2CD5MwZUZlZr/Slices-handoff?type=design&node-id=707-142993 - Connect your Figma account 

Definition of Done

Code complete

Code reviewed/PR approved

Design reviewed

Merged into MASTER

Available to view in Storybook.

Description

Build the UI for the new slice as per the design file.

Integration with TPA slice data to follow as a separate piece of work

Design: https://www.figma.com/file/phASjaJaTK2CD5MwZUZlZr/Slices-handoff?type=design&node-id=707-142993 - Connect your Figma account

[TMRX-1099]: https://nidigitalsolutions.jira.com/browse/TMRX-1099?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/ts-newskit/src/theme/times-web-light/style-presets/navigation.ts [hunk 1]
```diff
@@ -93,5 +93,10 @@ export const navStylePresets = {
     base: {
       backgroundImage: '{{overlays.overlayGradientBaseHorizontal}}'
     }
+  },
+  sectionBucketScrollOverlay: {
+    base: {
+      backgroundImage: 'transparent'
+    }
   }
```
Comment (douglasmik) [inline packages/ts-newskit/src/theme/times-web-light/style-presets/navigation.ts:101]: quite generic, thinking other components might bee able to use this preset, so might be better to have a generic name?
e.g. `transparentBackground`

**Review by douglasmik — APPROVED** [review]

File: packages/ts-newskit/src/slices/section-bucket/__tests__/index.test.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    />
+  );
+
+describe('Render SectionBucket Slice', () => {
+  test('Slice matches snapshot with sm', () => {
+    (useBreakpointKey as any).mockReturnValue('sm');
+    const { asFragment } = renderComponent();
+    expect(asFragment()).toMatchSnapshot();
+  });
+  test('Slice matches snapshot with xl', () => {
```
Comment (bibicollins1) [inline packages/ts-newskit/src/slices/section-bucket/__tests__/index.test.tsx:34]: Could we get some more declarative testing here?

File: packages/ts-newskit/src/theme/times-web-light/style-presets/navigation.ts (hunk 1 above)
Comment (ibrahim-kurhan) [reply packages/ts-newskit/src/theme/times-web-light/style-presets/navigation.ts:101]: you are right changed sectionBucketScrollOverlay to transparentBackground

File: packages/ts-newskit/src/slices/section-bucket/__tests__/index.test.tsx (hunk 2 above)
Comment (ibrahim-kurhan) [reply packages/ts-newskit/src/slices/section-bucket/__tests__/index.test.tsx:34]: added more test cases

**Pull request merged**